eelbrain/_io/stc_dataset.py

```diff
@@ -0,0 +1,155 @@
+import os
+import re
+import glob
+import itertools
+from mne import read_source_estimate
```

Comment:
```suggestion

from mne import read_source_estimate
```
Separate standard library and 3rd party imports